docs/getting-started.md

```diff
@@ -178,12 +178,13 @@ Generating Application spec manifest...
 Applying manifests to the cluster...
 Writing manifests to disk...
 Commiting and pushing wego resources for application...
-Pushing app manifests to repository...
 ```
 
 (*If the final lines are different, then most likely you have a problem with the SSH key used to deploy.*)
 
-8. Wait for the workload to show up in the cluster:
+8. A Pull Request has been created that has been set to merge into the branch set by --branch (default main). Set flag --auto-merge=true to skip this step.
```

Comment:
Ditto from above. Might also be a bit simpler to just say, "A Pull Request has been created against the branch..."